newa/__init__.py

```diff
@@ -382,12 +399,31 @@ def from_errata_tool(cls, event: Event) -> list[Erratum]:
 
 @define
 class Issue(Cloneable, Serializable):
```

Comment:
I think I will keep an eye on this class because I am not sure we really need it. My understanding is that for any operation with Jira we would be using classes from the `jira` module and I am wondering if there would be cases where we end up only with `Issue` class without getting the `JiraIssue` class right away (e.g. in order to add a comment etc.). I am not saying we should drop it right away, just keep it in mind as an opportunity to reduce "confusion".

Reply:
It actually make sense to keep it IMO. The reason is that we often only need to work with Jira ID (key, eg. NEWA-123) and for that having an envelope object (Issue) is kinda handy. When needed the actual content can be loaded into into (as it is currently implemented).